packages/amplify-category-api/src/graphql-transformer/utils.ts

```diff
@@ -6,7 +6,7 @@ import rimraf from 'rimraf';
 import {
   $TSContext, AmplifyCategories, CloudformationProviderFacade, JSONUtilities, pathManager, stateManager,
 } from 'amplify-cli-core';
-import { CloudFormation, Fn } from 'cloudform';
+import { CloudFormation, Fn } from 'cloudform-types';
```

Comment:
The package cloudform is all the same as cloudform-types except for an additional JSON.stringfy for the template. https://github.com/bright/cloudform/blob/master/packages/cloudform/index.ts
There are no difference when importing the types from either `cloudform` or `cloudform-types`

Reply:
Did we consider replacing cloudform all together with CDK?

Reply:
For the node18 migration we did not consider the option. The blocker is already resolved by replacing the `cloudform` with `cloudform-types`. Most of the cloudform code stays at v1 transformer and replacing all of them will take much implementation/testing effort.